### DISCUSSION TITLE
Comments in languages other than js

### DISCUSSION THREAD

**Discussion by obgibson** [code-hike/codehike#127] [Q&A] [answered]
Are label comment annotations supposed to work in all languages?

If I have

```js
// label something something
var a = 5;
```

I get a label

but if I try python, I do not

```python
 # label something something
 a = 5
```

**Comment by pomber** [comment, marked as answer]
Yes, but not yet. You can follow this issue #109

**Reply by obgibson** [reply]
Thanks Rodrigo!
